### DISCUSSION TITLE
Problem with the new version

### DISCUSSION THREAD

**Discussion by aaridoms** [JanDeDobbeleer/oh-my-posh#4680] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I just upgraded with scoop to the latest version and everything broke.

### Theme

I was using spaceship but changed to test if it was a problem with the theme.

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
It: Cannot process argument transformation on parameter 'test'. Cannot convert the "use" value of type "System.String" to type "System.Management.Automation.ScriptBlock".
Invoke-Expression: C:\Users\user\Documents\PowerShell\Microsoft.PowerShell_profile.ps1:11
Line |
  11 |  … -config "$env:POSH_THEMES_PATH\agnoster.omp.json" | Invoke-Expression
     |                                                        ~~~~~~~~~~~~~~~~~
     | Missing argument in parameter list.
```

**Comment by aaridoms** [comment]
![Captura de pantalla 2024-02-12 204433](https://github.com/JanDeDobbeleer/oh-my-posh/assets/28603433/198fefe6-ddac-4bff-9a24-0dfecf9fc348)

**Comment by JanDeDobbeleer** [comment]
@aaridoms what do you have in your `$profile` for oh-my-posh?

**Reply by JanDeDobbeleer** [reply]
Just validated the latest build and that seems fine.

**Reply by NathanielJS1541** [reply]
I'm having exactly the same issue since upgrading to oh-my-posh V19.9.0 with scoop.
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/33491104/396fb229-8fd4-463a-99e6-29db59748c06)
All I have in my profile is `oh-my-posh init pwsh | Invoke-Expression`.

It seems that `oh-my-posh` doesn't respond properly to the `--version` flag either, it just prints the help:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/33491104/c210d3ac-b82a-4c02-acb8-b08c9dd675d3)

**Reply by aaridoms** [reply]
apart from some functions and aliases

`oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH\agnoster.omp.json" | Invoke-Expression
`

**Reply by NathanielJS1541** [reply]
@aaridoms did you install `oh-my-posh` with `scoop`?

**Reply by aaridoms** [reply]
yes, i did

**Reply by JanDeDobbeleer** [reply]
@aaridoms confirmed to be broken on Windows, other OS'es work fine. No idea what's causing this tbh.

**Reply by JanDeDobbeleer** [reply]
[Reverting](https://github.com/JanDeDobbeleer/oh-my-posh/pull/4681) the upgrade to go1.22 until I figure out why that upgrade breaks Cobra.

**Reply by aaridoms** [reply]
Thank you Jan!

**Comment by Giresharu** [comment]
same issue here:
my $profile:
```shell
oh-my-posh init pwsh --config "E:\test.json" | Invoke-Expression
```

**Comment by JanDeDobbeleer** [comment]
[Reverting](https://github.com/JanDeDobbeleer/oh-my-posh/pull/4681) the upgrade to go1.22 until I figure out why that upgrade breaks Cobra.

**Reply by goldjg** [reply]
HugOps - thanks for jumping on this so quickly!

**Reply by goldjg** [reply]
This also prompted me to do something I should have done a while ago - sponsor you! You rock!

**Reply by goldjg** [reply]
manually installed v19.10.0 and all fixed (don't know how often Winget updates, it was still only offering 19.9.0)
Thank you @JanDeDobbeleer!

**Reply by lgtelles** [reply]
Just had the same issue here, will move to the version indicated above! :)

**Reply by deasems** [reply]
Thanks for the fix, force installing from winget didn't work for me, following the manual install instructions did.

**Reply by glav** [reply]
Also confirming, that running manual installation seemed to fix it. Using winget didn't do anything. It looked like the oh-my-posh dir in 
C:\Users\{user}\Documents\PowerShell\Modules\oh-my-posh was missing/empty.

Set-ExecutionPolicy Bypass -Scope Process -Force; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://ohmyposh.dev/install.ps1'))

**Reply by JanDeDobbeleer** [reply]
The correct version is available using winget now as well.

**Reply by nowhereman1999** [reply]
Hi, just wanted to add that I'm also encountering the "Invoke-Expression: Missing argument in parameter list" error on PowerShell 7.5.1 with the latest version of Oh My Posh.
![image](https://github.com/user-attachments/assets/761512a2-24e3-4968-bf59-68476430b773)
![image](https://github.com/user-attachments/assets/55979c96-7b2f-4795-a640-ff6b7f9e3665)
This issue occurs for me whether I install/reinstall Oh My Posh using winget or by using the official manual installation script (install.ps1). The error persists at startup.Previously, on PowerShell version 7.2.x, I did not encounter this issue at startup.here is my profile :
`oh-my-posh init pwsh --config $env:POSH_THEMES_PATH/jandedobbeleer.omp.json | Invoke-Expression`

**Reply by JanDeDobbeleer** [reply]
@Hezhexi2002 I'm pretty sure that error does not originate from oh-my-posh as otherwise the prompt would not be initialized.
